CHANGELOG.rst

```diff
@@ -4,6 +4,11 @@ Change history for XBlock SDK
 
 These are notable changes in XBlock.
 
+1.0.0 - 2024-05-30
```

Comment:
@farhan The version in `workbench/__init__.py` is 0.12.0, and that is what matters to PyPI, so everything else should match that. Please fix the changelog to say 0.12.0, and then publish a 0.12.0 release.

Reply:
@kdmccormick I have created the [PR](https://github.com/openedx/xblock-sdk/pull/391)
My point was we had dropped support of Python 3.8 now which means we should upgrade the major version. 
So should we update version to `1.0.0` now or should keep it below 1 yet?